composer.json

```diff
@@ -18,7 +18,8 @@
         "symfony/filesystem":  "~3.0|~4.0",
         "symfony/translation": "~3.0|~4.0",
         "symfony/form":        "~3.0|~4.0",
-        "symfony/asset":  "~2.8|~3.0|~4.0"
+        "symfony/asset":  "~2.8|~3.0|~4.0",
+        "symfony/console": "^4.1"
```

Comment:
Please also support `4.0` with `~4.0`. No reason not to.

Reply:
Ignore, I'll fix this in https://github.com/umpirsky/Twig-Gettext-Extractor/pull/61